spec/services/device_cleanup_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe DeviceCleanup do
+  describe "#run" do
+    it "will import feedback from Rpush" do
+      allow(Rpush).to receive(:apns_feedback)
+
+      DeviceCleanup.run
+
+      expect(Rpush).to have_received(:apns_feedback)
+    end
+
+    it "will delete devices from feedback table" do
+      device = create(:device)
+      another_device = create(:device)
+      Rpush::Apns::Feedback.create!(
+        device_token: device.token,
+        failed_at: Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.